servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultLoadBalancer.java

```diff
@@ -212,7 +212,7 @@ private static <R, C extends LoadBalancedConnection> long nextResubscribeTime(
         final long upperNanos = config.healthCheckResubscribeUpperBound;
         final long currentTimeNanos = config.executor.currentTime(NANOSECONDS);
         final long result = currentTimeNanos + (lowerNanos == upperNanos ? lowerNanos :
-                ThreadLocalRandom.current().nextLong(lowerNanos, upperNanos));
+                RandomUtils.nextLongInclusive(lowerNanos, upperNanos));
```

Comment:
is it ok that it went from exclusive to inclusive?

Reply:
I think so. See full comment on RRLB.